packages/yourloops/test/integration/assert/profile.ts

```diff
@@ -116,3 +118,33 @@ export const checkPatientProfilePage = (country: CountryCodes): PatientFieldsHtm
 
   return inputs
 }
+
+export const checkPasswordChangeRequest = async (): Promise<void> => {
+  const changePasswordCategoryTitle = screen.getByText('Security')
+  expect(changePasswordCategoryTitle).toBeVisible()
+
+  const changePasswordInfoLabel = screen.getByText('By clicking this button, you will receive an e-mail allowing you to change your password.')
+  expect(changePasswordInfoLabel).toBeVisible()
+
+  const changePasswordButton = screen.getByRole('button', { name: 'Change password' })
+  expect(changePasswordButton).toBeVisible()
```

Comment:
For buttons I like testing them with toBeEnabled / Disabled instead of just checking if they are visible

Reply:
I thought of that but given that this button is never disabled, I chose not to do it... Do you think it's relevant?